algolia/client.go

```diff
@@ -8,14 +11,17 @@ import (
 )
 
 var (
-	tmpIndex  = "libraries.tmp"
-	prodIndex = "libraries"
+	tmpIndex  = "libraries.tmp" // temp Algolia index
```

Comment:
Actually I can give more context here, can you add it as a comment:

The Algolia temporary index was used by the previous bot and I just copied it over because I didn't know how it worked. I don't see the value of building a temporary index and at the end renaming it to the production index. I would rather maintain a single index and only update entries when the bot pushed a new update.

--- 
that's not something we should do now, does that make sense @MattIPv4 ? If we do it live from the bot we could run into urls that 404 but that will go away with KV!

Reply:
Makes sense to me -- temp was used before so we only updated the prod index when it was all done, but yeah, kv should remove the need for that.